dagger/dagger.nim

```diff
@@ -62,6 +62,13 @@ proc new*(T: type DaggerServer, config: DaggerConf): T =
     .withTcpTransport({ServerFlags.ReuseAddr})
     .build()
 
+  # Most Local > Most Remote: order is important!
+  var stores: seq[BlockStore]
```

Comment:
Is this still needed?

Reply:
Probably not, will take another look.

Reply:
Updated so that the init block instantiates the size of the cache based on cli option.